assessments/projects/custodian/joint-review.md

```diff
@@ -0,0 +1,900 @@
+
+## Reviewers and Participants
+
+* Security Review Lead:
+* Security Co-Lead:
+* Security Co-Lead:
+* Security Reviewer:
+* Security Reviewer:
+* Project Lead:
+* Project Member:
+* Project Member:
+
+
+### Security links
+
+| Item | Ref or URL |
+| -- | -- |
+| Security Mailing List  | security@cloudcustodian.io |
+| Threat Model (2019) | https://docs.google.com/document/d/1S9zQZaT6G1TA3IAx6YNL0f7G938xaFZ-bziszhuxMZg |
+
+## Overview
+
+Cloud Custodian is a YAML DSL-based stateless rules engine for cloud auditing,
+management, and governance.
+
+Cloud Custodian’s policy-as-code:
+
+* Enables deployment of detection controls that enforce or report compliance to
+  organizational policies.
+* Supports real-time detection, reporting/notification, and remediation via an
+  API audit trail integration and serverless runtime support.
+* Provides consistent outputs and telemetry (blob, logs, trace, metrics) across
+  policies with outputs to provider native services.
+* Requires minimal installation requirements, and can be used as a CLI
+  query/investigative or operations tool in a compliance-as-code environment.
+
+### Background
+
+Cloud Custodian is a YAML DSL-based stateless rules engine operated via a CLI,
+easily triggered from a cloud-resident cron task or containerized workload.  It
+supports the practice of policy-as-code — policy being expressed as rules that
+the CLI uses to check cloud configurations.  The code artifacts serve as
+documentation of the configuration policies in effect in a given environment.
+
+NOTE: Humans must map written compliance or security documented policies to
+specific Cloud Custodian YAML first.
+
+At the moment, Cloud Custodian supports the most common public cloud providers:
+AWS, Azure, and GCP. Executed against these environments, with the
+corresponding YAML rules, Cloud Custodian can check cloud settings via cloud
+APIs for encryption and access requirements, tagging policies, cost management,
+removal of abandoned resources, and automated management of resources during
+off-hours.
+
+Cloud Custodian currently has [prototyped Kubernetes cluster support as
+documented here](https://github.com/cloud-custodian/cloud-custodian/pull/2760).
+
+### Project Goals
+
+Cloud Custodian integrates with the cloud APIs of each provider to provide
+checks for enforcement of resource provisioning and configuration policy goals.
+It can be run as a simple cron job on a server to execute against existing
+resources.  Cloud Custodian is essentially a rules engine for managing public
+cloud accounts and resources. It allows users to define custom yaml policies as
```

Comment:
nit: YAML v. yaml should be consistent throughout the doc.

Reply:
the official YAML docs use all caps YAML, so made consistent